justl.el

```diff
@@ -90,6 +95,12 @@
   :type 'integer
   :group 'justl)
 
+(defcustom justl-process-filter
```

Comment:
Do we need this anymore ? Since your implementation now handles both, I'm assuming it's not required anymore.

Reply:
I'd say it's nice to have customization point. For instance it's not impossible to use `compilation-filter` as I originally showcased and someone might have some specific hook for compilation-filter.

Reply:
I would want to avoid customization unless it's really required.

> For instance it's not impossible to use compilation-filter as I originally showcased and someone might have some specific hook for compilation-filter.

In the context of justfiles and it's execution, do you see any reason why someone might want to provide their own hook ?

Reply:
Except for the point above I don't have explicit example. I'll remove the customization point then.